Product

```diff
@@ -0,0 +1,27 @@
+class Product: 
+    def __init__(self, sale_price, number_products, purchase_price, name, brand, image, barcode, instructions, 
+        size):
+        self.sale_price = sale_price                # needs to be an integer
+        self.number_products = number_products      # needs to be an integer
+        self.purchase_price = purchase_price        # needs to be an integer
+        self.name = name                            # needs to be a string
+        self.brand = brand                          # needs to be a string 
+        self.image = image                          # needs to be a string
+        self.barcode = barcode                      # needs to be boolean 
+        self.instructions = instructions            # needs to be a string 
+
+
+    def design(): 
+        """ We will need certain sizes, materials and colors. """
+        
+        return ('We determine the size, texture and appropiate features the product should have to catch the attention of the users')
+
+
+    def get_price_product():  
```

Comment:
you can avoid `product` in the signature of the method because is kind of duplication.